content/docs/tutorials/fullstack.md

```diff
@@ -35,20 +35,225 @@ In this tutorial, we will be going through the entire process of creating a full
+```
+
+{{<break>}}
+
+## Create Database Connection
+For this project we will be utilizing a relational database and the SqlAlchemy ORM.  An in-depth example of how to set up SqlAlchemy ORM can be found in the examples section of our website as a part of the ["Integrate RelationalDBs"](https://cdevframework.io/docs/examples/relationaldb/#sqlalchemy-orm)chapter. 
+
+Update your `src/hello_world/resources.py` file to: 
+{{<codesnippet `/source_code/diary_tutorial/step1_resource.py`>}}
+
+Now we can deploy our resources.
+```bash
+cdev deploy
+```
+
+{{<break 1>}}
+**Next, we are going to create our models.** Install `sqlalchemy_aurora_data_api`.
+
+```bash
+pip install sqlalchemy_aurora_data_api
+```
+Then, create a `src/hello_world/models.py` file and add the following code:
+
+{{<codesnippet `/source_code/diary_tutorial/models.py`>}}
+
+Now we are going to install and configure `alembic`.
+
+```bash
+pip install alembic
+```
+{{<tool_tip key="tip" summary="Alembic Library">}}
+`Alembic` is one of the best in class tools for working with SqlAlchemy. You can learn more about the tool from [their official documentation](https://alembic.sqlalchemy.org/en/latest/).
+{{</tool_tip>}}
+
+Initialize the needed files for `alembic` using the following command. We will need to edit some of the generated files to connect to our db.
+```bash
+alembic init src/alembic
+```
+Replace the code in your `src/alembic/env.py` with the following:
+{{<codesnippet `/source_code/diary_tutorial/alembic.py`>}}
+
+Now to make sure our values are registered in the `env.py` script, we need to set our environments. 
+
+```bash
+export SECRET_ARN=$(cdev output --value hello_world_comp.relationaldb.demo_db.secret_arn)
+```
+```bash
+export CLUSTER_ARN=$(cdev output --value hello_world_comp.relationaldb.demo_db.cluster_arn)
+```
+```bash
+export DB_NAME=<db_name>
+```
+{{<break 1>}}
+
+You should now be able to create automated migrations. 
+
+{{<tool_tip key="warning" summary="Auto Generation limits">}}
+You should familiarize yourself with the [limits of alembic auto generation](https://alembic.sqlalchemy.org/en/latest/autogenerate.html#what-does-autogenerate-detect-and-what-does-it-not-detect) and **ALWAYS** confirm the changes before applying them.
+{{</tool_tip>}}
+
+```bash
+alembic revision --autogenerate -m "Added entries table"
+```
+Apply the migration with the `upgrade` command. This will create our `Entry` Table.
+```bash
+alembic upgrade head
+```
+
+{{<break 1>}}
+Lets now connect to our DB and add an User.
+```bash
+cdev run relationaldb.shell hello_world_comp.demo_db
+```
+```sql
+BEGIN
+```
+```sql
+INSERT INTO entries(title, content) VALUES ('test entry','test our db connection');
+```
+```sql
+COMMIT
+```
+```bash
+quit
+```
+
+## Create Serverless Functions
+Now that the database is set up, it is time to add some serverless functions.  We can start by uncommenting lines 5-6, 13, 43-44, and 53-58 of your `src/hello_world/resources.py` file.
```

Comment:
I think we should separate this into two files instead of using comments. Makes it faster to just copy and paste the whole file, but I could also see the benefit of them explicit turning on those sections. I just worry about them messing up the uncommenting. 

Thoughts?

Reply:
My thoughts were the user would probably benefit more from turning these sections on so that they see exactly what is being added and they will be more likely to understand the changes.  It definitely would be faster for them to copy and paste the entire file though, they just may not learn as much from it because they would have to inspect the files to find the differences, or we would have to write out what is being added and explain it in writing.  Not sure which route is best.

Reply:
I like the commented lines. The ones inside the function are great and I can tell I would appreciate having it right there.
If we have the instructions, I don't think it would be a problem for the user change the code and it could help them get more familiar with it.